website/docs/docs/build/data-tests.md

```diff
@@ -306,6 +304,8 @@ data_tests:
 
 </File>
 
+To suppress warnings about the rename, add `TestsConfigDeprecation` to the `silence` block of the `warn_error_options` flag in `dbt_project.yml`, [as described in the Warnings documentation](https://docs.getdbt.com/reference/global-configs/warnings).
```

Comment:
We can change this warning name once this issue is addressed https://github.com/dbt-labs/dbt-core/issues/10504

Reply:
[I checked here](https://dbt-labs.slack.com/archives/C05FWBP9X1U/p1722504680313799?thread_ts=1722303179.086539&cid=C05FWBP9X1U), and the TestConfigDeprecation name itself won't change, just the warning message it returns. (Because changing that name would itself be a breaking change!)